src/Post.php

```diff
@@ -28,14 +31,29 @@ class Post extends AbstractEndpoint {
 	 * @return array|\WP_Error
 	 */
 	public function endpoint_callback( \WP_REST_Request $request ) {
-		$slug = trim( $request->get_param( 'slug' ), '/' );
+		$params = $request->get_params();
```

Comment:
`get_params` combines the default args with the ones passed by the user so by set other values rather than false you can get defined values on the array.